src/HttpServer.ts

```diff
@@ -113,11 +136,56 @@ export class HttpServer {
         }
     }
 
-    statusHandler(req: any, res: any): void {
+    /**
+     * @openapi
+     * /status:
+     *   get:
+     *     summary: It returns a 204 response with an empty body.
+     *     description: It may be used just to check if the server is running.
+     *     responses:
+     *       '204':
+     *         description: No Content.
+     */
```

Comment:
this is pretty awful (the function, not the documentation).

i suggest creating an issue to work on these endpoints, an empty `204` as a ping is far from ideal

Reply:
Create [a task](https://rsklabs.atlassian.net/browse/PP-126) for this.